common/app/mvt/MultiVariateTesting.scala

```diff
@@ -47,7 +47,7 @@ object ABNewDesktopHeaderControl extends TestDefinition(
   sellByDate = new LocalDate(2017, 8, 24)
 ) {
 
-  def participationGroup(implicit request: RequestHeader): Option[String] = request.headers.get("X-GU-ab-new-desktop-header")
+  def participationGroup(implicit request: RequestHeader): Option[String] = request.headers.get("X-GU-ab-new-desktop-header-v2")
```

Comment:
We don't for just opting in, but we will need it soon again for v2 so I thought I might as well keep it.